Tasks/DownloadSecureFileV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -7,4 +7,4 @@
   "loc.input.help.secureFile": "The file name or GUID of the secure file to download to the agent machine. The file will be deleted after the pipeline runs.",
   "loc.input.label.retryCount": "Retry Count",
   "loc.input.help.retryCount": "Optional number of times to retry downloading a secure file if the download fails."
-}
+}
```

Comment:
?

Reply:
Same as above - autogenerated by the build/maybe git, I think someone probably had bad line endings when they checked this one in